app/styles/_mixins.scss

```diff
@@ -84,6 +84,20 @@
   }
 }
 
+@mixin mouse-state-colors($color-hover, $color-focus, $color-active) {
```

Comment:
If I was going to be a pedant here, I'd say that calling it "mouse" state seems wrong here because `focus` and `active` can both be achieved without a mouse. Would calling it `element-state-colors` too generic?

But anyway, I'm not going to be a pedant here, so I won't say any of the above.